A tutorial on how to create a 3Box chatbox plugin.md

```diff
@@ -0,0 +1,11 @@
+# Content
+A tutorial on how to create a 3Box chatbox plugin
+
+Outline 
+
+1). An overview of 3box chatbox plugin
+2). Introduction to 3Box chatbox plugin
+3). Use cases of 3Box chatbox plugin
+4). How 3box chatbox works
+(i) example of 3box chatbox plugin
+5). How to build a 3box chatbox plugin.
```

Comment:
@writeprovidence What's the difference between points 4) i. and 5?

Reply:
4 i) Will be making reference to an already existing chatbox plugin, while (5) will be detaining the process of building a 3box chatbox plugin.

Reply:
OK, sounds good.